bot/resources/tags/docstring.md

```diff
@@ -1,18 +1,16 @@
-A [`docstring`](https://docs.python.org/3/glossary.html#term-docstring) is a string with triple quotes that often used in file, classes, functions, etc. A docstring should have a clear explanation of exactly what the function does. You can also include descriptions of the function's parameter(s) and its return type, as shown below.
+A [`docstring`](https://docs.python.org/3/glossary.html#term-docstring) is a string with triple quotes that's placed at the top of files, classes and functions. A docstring should contain a clear explanation of what it's describing. You can also include descriptions of the subjects's parameter(s) and its return type, as shown below.
 ```py
-def greet(name, age) -> str:
-  """
-  Return a string that greets the given person, including their name and age.
+def greet(name: str, age: int) -> str:
+    """
+    Return a string that greets the given person, including their name and age.
 
-  :param name: The name to greet.
-  :type name: str
-  :param age: The age to display.
-  :type age: int
-  :return: String of the greeting.
-  """
-  return_string = f"Hello, {name} you are {age} years old!"
-  return return_string
+    :param name: The name to greet.
+    :param age: The age to display.
```

Comment:
```suggestion
    :param age: The person's age.
```

Reply:
I was actually about to commit `The age of the person to include in the greeting.` What do you think?

Reply:
Something more like `The age of the person to greet`? Since we want to know the age of the person that we're about to greet, the person isn't necessarily *included*.